pkg/cmd/roachtest/main_test.go

```diff
@@ -233,12 +420,73 @@ func getTestSelectionMockData() ([]registry.TestSpec, *sqlmock.Rows) {
 		{"t_randomized", "no", "5", "no"},
 	}
 	// rows are the rows that are returned by snowflake. The list of string represents the columns of each row
-	rows := sqlmock.NewRows([]string{
-		"name", "selected", "avg_duration", "last_failure_is_preempt",
-	})
+	rows := sqlmock.NewRows(testselector.AllRows)
 	// rows are populated with the data
 	for _, ds := range data {
 		rows.FromCSVString(strings.Join(ds, ","))
 	}
 	return specs, rows
 }
+
+const charset = "abcdefghijklmnopqrstuvwxyzABCDEFGHIJKLMNOPQRSTUVWXYZ0123456789"
+
+// generateRandomString generates a random string of a given length
+func generateRandomString(n int) (string, error) {
```

Comment:
Same as above:

`randutil.RandString`

Reply:
This may not work as the length variation cannot be achieved.